src/main/groovy/org/mockito/release/internal/gradle/DefaultContinuousDeliveryPlugin.java

```diff
@@ -315,11 +315,17 @@ public void execute(Task task) {
     private static void configureNotableReleaseNotes(Project project) {
         VersionFile versionFile = project.getExtensions().getByType(VersionFile.class);
         NotableReleaseNotesGeneratorTask task = (NotableReleaseNotesGeneratorTask) project.getTasks().getByName("updateNotableReleaseNotes");
+        NotableNotesFetcherTask fetcherTask = (NotableNotesFetcherTask) project.getTasks().getByName("fetchNotableNotes");
+
```

Comment:
Let's keep naming consistency, for clarity. For example:
"updateNotableReleaseNotes + fetchNotableReleaseNotes", "updateNotableNotes + fetchNotableNotes".